lib/fortnox/api/types.rb

```diff
@@ -38,6 +38,15 @@ module Types
       AccountNumber = Strict::Int
                       .constrained(gteq: 0, lteq: 9999)
                       .optional
+                      .constructor do |input|
```

Comment:
This constructor is now 9 lines. Can we create a separate method for this? I'm pretty sure Rubocop will complain about this :)